spec/lib/stripe/account_connector_spec.rb

```diff
@@ -15,7 +15,7 @@ module Stripe
       let(:connector) { AccountConnector.new(user, params) }
 
       before do
-        allow(Stripe).to receive(:api_key) { "sk_test_12345" }
+        Stripe.api_key = "sk_test_12345"
```

Comment:
Could we just set all these things once in `spec_helper.rb` or `environments/test.rb`, and drop all this code? Eg:
```ruby
Stripe.api_key = "sk_test_12345"
Stripe.client_id = "ca_abc123"
Stripe.publishable_key = "some_token"
```

Does that work? Is it preferable?

Reply:
Good thinking. Let's find out :) 4e7adb7d4c28092029f4874eed026cf6f8b791c4

Reply:
For some reason I lost this commit, I had to replay it manually :-) Done, will see the build now.

Reply:
ok, as far as I can see, it doesnt work, we just get all test broken again with "No API key defined"

Reply:
for reference, the commit is here: https://github.com/luisramos0/openfoodnetwork/commit/dfcf311149a2e5bc54d8a274b03ccdce479834c1

Reply:
I had another quick try at this and didn't come to an immediate solution. IMO this is more of a nice-to-have and shouldn't hold up the whole PR from going to Test Ready - the lack of DRYness already existed and is unrelated to Rails 6. What do you think @Matt-Yorkley?